### PR TITLE
Add a link to the list of supported events

### PR DESCRIPTION
**Pull request by scjody**
Just a quick doc update - it took us a while to find this information.

### CODE REVIEW

File: docs/docs/10.4-test-utils.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Simulate.{eventName}(
 )
 ```
 
-Simulate an event dispatch on a DOM node with optional `eventData` event data. **This is possibly the single most useful utility in `ReactTestUtils`.**
+Simulate an event dispatch on a DOM node with optional `eventData` event data. **This is possibly the single most useful utility in `ReactTestUtils`.**R
```
Comment (zpao) [inline docs/docs/10.4-test-utils.md]: Can you remove this stray character?

**Comment by zpao** [comment]
Other than the character fix, looks good. Thanks! Can you make sure you squash any changes when you updated so we only have a single commit?

File: docs/docs/10.4-test-utils.md (hunk 1 above)
Comment (scjody) [inline docs/docs/10.4-test-utils.md]: @zpao Done.  Sorry I didn't notice that earlier.

**Comment by zpao** [comment]
Thanks!

**Pull request merged**